src/DataAPI.jl

```diff
@@ -73,4 +73,22 @@ definition.
 """
 function describe end
 
+"""
+    Between(first, last)
+Select the columns between `first` and `last` from a table.
```

Comment:
Sorry to be late, but I think the convention is to have an empty row after the signature, right?

Reply:
Right - I will add the fix (I copy pasted this from IndextedTables.jl, but now I see that it is correct there)